Makefile

```diff
@@ -23,9 +23,8 @@ RM=--rm
 RUN_FLAGS=-ti
 BUILD_IN_CONTAINER=true
 GO_ENV=GOGC=off
-GO_BUILD_INSTALL_DEPS=-i
```

Comment:
Do we want to drop `-i` from the instruction `echo "    sudo go clean -i net";` down below as well?

Reply:
`-i` is actually correct there, as the intention was to install a different build of the standard library, but the whole bit is irrelevant now that Go caches builds for different tags in different places, so I'll remove it.